### ISSUE TITLE
Cannot DISAMBIGUATE follow the hint

### ISSUE THREAD

**Issue by sihcpro** [PostgREST/postgrest#1518]
### Environment
* **Operating system**: MacOS -  Sierra - Version 10.12.6
* **PostgreSQL version**: 11.4 +
* **PostgREST version**: docker: postgrest/postgrest:v7.0.0
* **Python version**: Python 3.7.7
* **Platform**: Sanic
* **Do you use pgbouncer?**: No
* **Did you install with pip?**: Yes

### Description of issue

I have a table `member` in schema `A`
I also have a table `user` in schema `B`
There are a foreign key between `member` and `user` on `member.npi` and `user.npi`. It's name is `npi_user_fk` and save on table `member` in schema `A`.
And I create a full view (SELECT *) from table `member` in schema `A` into schema `B`

And when I try to embed `user` into `member`, there are some hint that I don't understand:

With the following link: [http://0.0.0.0:5033/member?select=user(*)](http://0.0.0.0:5033/member?select=user(*))
I got:
```json
{
    "hint": "By following the 'details' key, disambiguate the request by changing the url to /origin?select=relationship(*) or /origin?select=target!relationship(*)",
    "details": [
        {
            "relationship": "npi_user_fk[npi][npi]",
            "origin": "B.member",
            "cardinality": "m2o",
            "target": "B.user"
        },
        {
            "relationship": "A.encounter[encounter-member-id-fk][encounter-user-id-fk]",
            "origin": "B.member",
            "cardinality": "m2m",
            "target": "B.user"
        },
        {
            "relationship": "A.encounter-view[encounter-member-id-fk][encounter-user-id-fk]",
            "origin": "B.member",
            "cardinality": "m2m",
            "target": "B.user"
        },
        {
            "relationship": "A.member--attribution[member--attribution_member_key_fkey][member--attribution_npi_fkey]",
            "origin": "B.member",
            "cardinality": "m2m",
            "target": "B.user"
        }
    ],
    "message": "More than one relationship was found for member and user"
}
```

With the following link: [http://0.0.0.0:5033/member?select=npi_user_fk(*)](http://0.0.0.0:5033/member?select=npi_user_fk(*))
I got:
```json
{
    "hint": "By following the 'details' key, disambiguate the request by changing the url to /origin?select=relationship(*) or /origin?select=target!relationship(*)",
    "details": [
        {
            "relationship": "npi_user_fk[npi][npi]",
            "origin": "B.member",
            "cardinality": "m2o",
            "target": "B.user"
        },
        {
            "relationship": "npi_user_fk[npi][npi]",
            "origin": "B.member",
            "cardinality": "m2o",
            "target": "B.user-contract-condition"
        },
        {
            "relationship": "npi_user_fk[npi][npi]",
            "origin": "B.member",
            "cardinality": "m2o",
            "target": "B.user-summary"
        }
    ],
    "message": "More than one relationship was found for member and npi_user_fk"
}
```

`user-contract-condition` and `user-summary` is two VIEW that also have foreign
 `npi`.
I just want to embed `user` TABLE to `member` VIEW. So, what should I do?

**Comment by bwbroersma** [comment]
Can you share the (minimum) SQL for the tables `A.member`, `B.user` and the view `B.member`?

Did you read the [Embedding Disambiguation](http://postgrest.org/en/v7.0.0/api.html#embed-disamb) in the manual?

PS. why are you mentioning python & pip in your PostgREST install environment?

**Comment by steve-chavez** [comment]
@sihcpro Also, include the PostgREST `db-schema` you're using(or `PGRST_DB_SCHEMA` if using docker).

**Comment by sihcpro** [comment]
Now I understand what is the problem:
Example:
Table C have column `b_id` and column `a_id` that contain `_id` of table B and `_id` of table A. There are 2 foreign keys are created: "a_fk" and "b_fk"
When I create a `VIEW` named D from table C, include columns `b_ic` and `a_id`. 2 foreign keys in table C are duplicated. And now we have 4 foreign keys. 2 "a_fk" and 2 "b_fk".
Now when I join table A into table B, there are 2 way to join (1 by table C, 1 by view D), and when I choose the foreign key name, there are still  vague. So I believe that that is why I can't embed table B into table A.

**Comment by steve-chavez** [comment]
@sihcpro [Hint disambiguation](http://postgrest.org/en/v7.0.0/api.html#hint-disambiguation) may help for that case(since it involves VIEWs).

The text description is a bit hard to follow, we could be more helpful if you shared the DDL(CREATE TABLE, CREATE VIEW).

**Comment by steve-chavez** [comment]
@sihcpro Did my suggestion help you to solve the issue?

**Comment by steve-chavez** [comment]
Considering the issue resolved for now, please reopen if not.
